tests/db/assessmentsDB.test.js

```diff
@@ -0,0 +1,21 @@
+const testEnv = require('./test-environment')
+const db = require('../../server/db/assessmentsDb')
+
+beforeEach(() => {
+  testDb = testEnv.getTestDb()
```

Comment:
You haven't defined the variable `testDb` before this line, hence the error `testDb is not defined`.

Reply:
added let testDb = null at before line 4.